convert2rhel/systeminfo.py

```diff
@@ -154,6 +154,27 @@ def _generate_rpm_va(self):
         self.logger.info("The 'rpm -Va' output has been stored in the %s file"
                          % output_file)
 
+    def generate_modified_rpms(self):
+        """Generate a log and stdout message with modified rpms after the convert process """
```

Comment:
Should be a dot at the end of the sentence

Reply:
without space

Reply:
Not sure I like that the function both generates a log file as well as outputting changes to stdout, should be split

Reply:
This code was re-factored...

Reply:
@fellipeh you forgot to push the commit to GitHub